index.js

```diff
@@ -20,19 +28,55 @@ AFRAME.registerComponent('simple-link', {
   /**
    * Called once when component is attached. Generally for initial setup.
    */
-  init() {},
+  init() {
+    this.navigate = this.navigate.bind(this);
+    const { el } = this;
+
+    el.setAttribute('geometry', {
+      primitive: 'circle',
+      radius: 1
+    });
+    if (this.data.image) {
+      el.setAttribute('material', {
+        src:
+          typeof this.data.image === 'string'
+            ? this.data.image
+            : this.data.image.src,
```

Comment:
Hmm.. I wonder if this is needed? https://aframe.io/docs/0.8.0/core/component.html#schema_asset

I _think_ this documentation is indicating that you should be able to pass a URL string, or an ID. If you pass it an ID it'll run `getAttribute('src')` on the asset entity corresponding with the ID you pass in.